com.teamspeak.TeamSpeak.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this is the wrong icon. The actual app icon looks like this:

![com teamspeak teamspeak3_48](https://user-images.githubusercontent.com/18191412/51484513-0f482200-1d9c-11e9-934b-d2714adcffc7.png)

Reply:
I tried to find the icon and this was packaged with the application. Where did you find this icon?

Reply:
I extracted it from the macOS binary using 7z and icns2png

Reply:
@tilosp It seems that the macOS icon is different than the Linux one and the currently used icon is the right one.

Reply:
@AsciiWolf i don't think so. If i directly run the linux executable, it shows me the icon i posted above.
The other one is used as an background element in the gui.

Reply:
I looked at the teamspeak.com and they are using the same icon as I'm using in Flatpak.

Reply:
@tilosp Hmm, when I run the TeamSpeak binary directly, it has no icon.

Reply:
Windows, MacOS, [Andoid](https://play.google.com/store/apps/details?id=com.teamspeak.ts3client) and [iOS](https://itunes.apple.com/app/teamspeak-3/id577628510) all use similar app icons (blue background with white logo). Even their [youtube channel](https://www.youtube.com/channel/UCenDIW8PPnFDhu8_gkqcmrQ) and [twitter](https://twitter.com/teamspeak) uses this logo.

@Zlopez They use different colored logos on their website depending on the background.

But for an app icon this doesn't make sense because you don't know which color the background has. In the most extreme case the icon wouldn't even be visible because the background has the same color.

@AsciiWolf 
Which desktop environment are you running?
![screenshot from 2019-01-21 20-41-28](https://user-images.githubusercontent.com/18191412/51495658-68747d80-1dbd-11e9-9c47-a8e4297bb2a2.png)
![screenshot from 2019-01-21 20-42-02](https://user-images.githubusercontent.com/18191412/51495660-6ad6d780-1dbd-11e9-97d0-6066ee1f9a16.png)

Reply:
@tilosp GNOME Shell 3.30 on Fedora 29.

Reply:
The icon is now changed.